images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -11,12 +11,12 @@ source $HELPER_SCRIPTS/os.sh
 # Ubuntu 20 doesn't support EOL versions
 if isUbuntu20 ; then
     LATEST_DOTNET_PACKAGES=("dotnet-sdk-3.1")
```

Comment:
this one should change to 5.0 instead of 3.1

Reply:
5.0 package is currently not present. I guess these packages are not for prerelease versions. We will need to update this only after ga.